src/encoded/types/publication.py

```diff
@@ -175,14 +159,18 @@ def map_doi_pmid(doi):
 
 def map_doi_biox(doi):
     "If a doi is not mapped to pubmed, check where it goes"
-    DOIad = "https://doi.org/"
-    www = "{DOIad}{doi}".format(DOIad=DOIad, doi=doi)
+    DOIapi = "https://doi.org/api/handles/"
+    www = "{DOIapi}{doi}".format(DOIapi=DOIapi, doi=doi)
     for count in range(5):
         resp = requests.get(www)
         if resp.status_code == 200:
             break
-    landing_page = resp.url
-    if "biorxiv" in landing_page.lower():
+    if resp.json().get('values'):
+        for value in resp.json()['values']:
+            if value.get('type', '') == 'URL':
+                landing_page = value['data']['value']
+                break
+    if "biorxiv" in landing_page:
```

Comment:
No need to lowercase as before?

Reply:
No, this is now the URL to which doi.org redirects, and 'biorxiv' here is already lowercase.